app/src/main/res/layout/screen_patient_summary.xml

```diff
@@ -79,9 +45,43 @@
       android:paddingTop="2dp"
       android:text="@string/patientsummary_edit"
       android:textAppearance="@style/Clinic.V2.TextAppearance.Button2.White100"
-      tools:ignore="UnusedAttribute" />
+      tools:ignore="RelativeOverlap,UnusedAttribute" />
 
-  </LinearLayout>
+    <TextView
+      android:id="@+id/patientsummary_fullname"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_alignParentStart="true"
+      android:layout_below="@id/patientsummary_back"
+      android:layout_marginTop="@dimen/spacing_8"
+      android:textAppearance="@style/Clinic.V2.TextAppearance.PatientSummary.PatientName"
+      tools:text="Anish Acharya, M, 43" />
+
+    <TextView
+      android:id="@+id/patientsummary_address"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_alignStart="@id/patientsummary_fullname"
+      android:layout_below="@id/patientsummary_fullname"
+      android:layout_marginTop="@dimen/spacing_4"
+      android:textAppearance="@style/Clinic.V2.TextAppearance.Body2Left.White72"
+      tools:text="House No. 14, Street No. 10/11, Guru Gobind Singh Nagar, Bathinda, Punjab" />
+
+    <TextView
+      android:id="@+id/patientsummary_contact"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_alignStart="@id/patientsummary_address"
+      android:layout_below="@id/patientsummary_address"
+      android:layout_marginTop="@dimen/spacing_4"
+      android:drawablePadding="@dimen/spacing_4"
+      android:drawableStart="@drawable/ic_call_16dp"
+      android:drawableTint="@color/white72"
+      android:textAppearance="@style/Clinic.V2.TextAppearance.Body2Left.Numeric.White72"
+      android:visibility="gone"
+      tools:text="1111111111"
+      tools:visibility="visible" />
+  </RelativeLayout>
```

Comment:
@vinaysshenoy the spacings don't match with Zeplin. Can you check?

Reply:
Everything else looks good. Let me know once you've fixed this.

Reply:
Done.